articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+## Configure Azure
+
+::: note
+To complete this, tutorial, you will need to have an account that grants you access to Microsoft's [Azure Portal](https://portal.azure.com).
+:::
+
+In this section, you'll begin by creating an API management instance, as well as the API to manage your cookies application.
+
+### Create Your API Management Instance
+
+To create your API management instance, log into the Azure Portal.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/azure-portal-home.png)
+
+To create a new API management service, click on **New** > **Web + Mobile** > **API management**.
```

Comment:
I'd remove both the first sentence (`To create your API management instance, log into the Azure Portal.`) and the first screenshot. Change the second sentence to `To create a new API management service, log in the Azure Portal and click on **New** > **Web + Mobile** > **API management**.`